src/deepsparse/v2/schedulers/scheduler.py

```diff
@@ -37,6 +38,17 @@ class OperatorScheduler:
     def __init__(self, max_workers: int = 1):
         self._threadpool = ThreadPoolExecutor(max_workers=max_workers)
 
+    def async_run(
+        self, *args, operator: Operator, loop: Any, **kwargs
```

Comment:
If you're looking for a type for `loop` other than `Any`, see this comment: https://github.com/neuralmagic/deepsparse/pull/1380/files?diff=split&w=0#r1391476543